### ISSUE TITLE
Incorrect lcm result

### ISSUE THREAD

**Issue by sgerace** [compute-io/lcm#3]
I recently ran into a case where `lcm` was returning the incorrect result. The numbers 186028, 193052 and 144624, as computed by:

``` js
const lcm = require('compute-lcm');
console.log(lcm(186028, 193052, 144624));
```

reports the result 108206102374928, however the correct answer is 324618307124784.

**Comment by kgryte** [comment]
@sgerace Thanks for reporting. Was able to reproduce. Seems to stem from a bug in the [`gcd`](https://github.com/compute-io/gcd/issues/3) implementation.

A patch submitted against `gcd` should resolve the issue.

**Comment by kgryte** [comment]
This bug is not present in the [`stdlib`](https://github.com/stdlib-js/stdlib/tree/6ae64811e7dae8d8a54b5d0ebb6de38f1ca61068/lib/node_modules/%40stdlib/math/base/special/lcm) implementation.

We will, however, push a patch to `gcd`.

**Comment by kgryte** [comment]
This was fixed in [`gcd`](https://github.com/compute-io/gcd/commit/e475160f77f00e4b6cdc319d2b09296f4480839a). @sgerace Can you reinstall your `node_modules` dependencies and confirm that the issue is resolved?

**Comment by kgryte** [comment]
The `gcd` dep version has been [updated](https://github.com/compute-io/lcm/commit/929b766884c68eded7f19db62cc79cdbd79144d2), a test case added, and a new version published. Closing as resolved.

Thank you, @sgerace, for reporting and @Planeshifter for fixing.

**Comment by sgerace** [comment]
Hi @kgryte, thanks for the quick response. I can confirm that the issue has been fixed in 1.1.2.
